### PR TITLE
'toLowerCase' of undefined fixed on ap-southeast-2 region

### LINKED ISSUE #1387: NEW ap-southeast-2 Region Support
New Region just announced :  ap-southeast-2
please add

### PR DESCRIPTION
**Pull request by tengis**
bug fixes referenced on issue #1387

### CODE REVIEW

File: lib/Region.js [hunk 1]
```diff
@@ -73,8 +73,9 @@ module.exports = function(S) {
       if (name.includes('euwest1'))      return 'eu-west-1';
       if (name.includes('eucentral1'))   return 'eu-central-1';
       if (name.includes('apnortheast1')) return 'ap-northeast-1';
+      if (name.includes('apsoutheast2')) return 'ap-southeast-2';
```
Comment (tobyhede) [inline lib/Region.js]: That's the bit I was missing

Comment (tengis) [inline lib/Region.js]: @tobyhede Yep 😄

**Comment by jeffski** [comment]
Please merge and release soon. Looking forward to using Serverless in the Sydney region.

**Comment by rojotek** [comment]
I have just deployed a test serverless app to Sydney using this code change.  It would be great to see it become a first class citizen.

**Comment by flomotlik** [comment]
Hi @rojotek @jeffski @tobyhede @tengis , thanks for the contribution, but we've stopped working on 0.5 of Serverless at this point to fully focus on the next release with V1 that we're currently working on. Therefore we're not taking any commits for 0.5 at this point and I'm closing this PR. We only have a limited set of resources and just need to make sure we're putting everything we have into our next release (and can provide great long term support and backwards compatibility from then on). Hope you understand.

We would love to get your feedback on V1 of the framework and I'm happy to walk you through any questions to make sure its all working great for you.

**Pull request closed without merging**